src/components/CostEstimationContent/CostOperation.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { FunctionComponent, useContext, useState, useEffect, useCallback } from "react";
+import { OverlayContext } from "@govtechsg/tradetrust-ui-components";
+import { useFetchGasPrice } from "../../common/hooks/useFetchGasPrice";
+import { CostData, CostDataFn } from "./types";
+import { CostModal } from "./CostModal";
+import { currentDateStr } from "../../utils";
+
+export const CostOperation: FunctionComponent = () => {
+  const [dateTime, setDateTime] = useState(currentDateStr());
+  const { price, gwei } = useFetchGasPrice("ethereum", 30000);
+  const { price: maticPrice, gwei: maticGwei } = useFetchGasPrice("polygon", 30000);
+  const priceFactor = gwei * 0.000000001 * price;
+  const maticPriceFactor = maticGwei * 0.000000001 * maticPrice;
+
+  const transferHoldershipGas = 43634;
+  const transferOwnershipGas = 324688;
+  const issueDocGas = 239523;
+  const burnDocGas = 56532;
+  const surrenderDocGas = 93435;
+
+  useEffect(() => {
+    setDateTime(currentDateStr());
+  }, [gwei]);
```

Comment:
this says that only when ethereum gwei changes, then we set the date time, which means what you say in the description only applies to ethereum prices which is probably not correct.

This might be a design fault so need to clarify with jerry

Reply:
ok